addons/pos_restaurant/static/src/overrides/components/ticket_screen/ticket_screen.js

```diff
@@ -21,8 +21,8 @@ patch(TicketScreen.prototype, {
         if (table) {
             let floorAndTable = "";
 
-            if (this.pos.floors && this.pos.floors.length > 1) {
-                floorAndTable = `${table.floor.name}/`;
+            if (this.pos.models["restaurant.floor"].length > 0) {
```

Comment:
shouldn't this be the following?
```suggestion
            if (this.pos.config.floor_ids.length > 0) {
```

Reply:
I prefer to do it like that:

```
        if self.config_id.module_pos_restaurant:
            params.update({
                'restaurant.floor': {
                    'domain': [('pos_config_ids', '=', self.config_id.id)],
                    'fields': ['name', 'background_color', 'table_ids', 'sequence'],
                },
                'restaurant.table': {
                    'domain': lambda data: [('active', '=', True), ('floor_id', 'in', [floor['id'] for floor in data['restaurant.floor']])],
                    'fields': [
                        'name', 'width', 'height', 'position_h', 'position_v',
                        'shape', 'floor_id', 'color', 'seats', 'active'
                    ],
                }
            })

```

no floor and table will be loaded at all if we're not in restaurant mode.

Reply:
But to be explicit, I'll also use config.floor_ids.